platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationImpl.java

```diff
@@ -37,9 +40,9 @@ class ConfigurationImpl implements Configuration, ConfigLifecycle {
     private boolean initialized = false;
 
     ConfigurationImpl(
-            final ConfigPropertiesService propertiesService,
-            final ConverterService converterService,
-            final ConfigValidationService validationService) {
+            @NonNull final ConfigPropertiesService propertiesService,
```

Comment:
(Same comments in this file as previous comments w.r.t. `Objects.requireNonNull` and returning (or not) empty sets.)

Reply:
I did this PR before we had that discussion ;)